external/radiation/tests/test_driver.py

```diff
@@ -1,13 +1,18 @@
-from config import FORCING_DIR, LOOKUP_DIR, nlay
```

Comment:
Does pytest end up calling this when it collects tests? would be nice to wrap in a function so it acts like a normal pytest test.

Reply:
It does, assuming we add pytest to that nix environment. We can clean up the test interface.